### DISCUSSION TITLE
Apple Private Relay

### DISCUSSION THREAD

**Discussion by no0ID** [kkebo/DNSecure#50] [Q&A] [answered]
I need advice regarding the coexistence of Apple Private Relay and DNSecure.
If APR is enabled together with DNSec then I get listed as active DNS servers those enabled by APR (almost exclusively Cloudflare) and the server enabled in DNSecure. What to use when not using VPN?
I know that APR is not a VPN replacement, additionally the WebRTC leaks the original IP...

**Comment by kkebo** [comment, marked as answer]
I believe using APR is superior to enabling DoH/DoT using DNSecure. APR is not only easy to use but also uses the Oblivious DoH even though DNSecure can't support it.

> DNS servers those enabled by APR (almost exclusively Cloudflare)

APR's DoH server is depending on what Egress Proxy you are using at the time.

APR consists of the Ingress Proxies and the Egress Proxies, which are provided by Akamai, Fastly, and Cloudflare. Your network route will be randomly selected from those vendors and will change at regular intervals. Depending on which vendor is selected, various behaviors (e.g. DNSSEC, IPv6) will be different.
